src/EVM.hs

```diff
@@ -230,6 +230,17 @@ isCreation = \case
 next :: (?op :: Word8) => EVM t s ()
 next = modifying (#state % #pc) (+ (opSize ?op))
 
+getOpName :: forall (t :: VMType) s . FrameState t s -> [Char]
+getOpName state = intToOpName $ fromEnum $ (?op)
```

Comment:
```suggestion
getOpName state = intToOpName $ fromEnum (?op)
```

I also don't understand why do you call it `?op`.
From what I read question mark at the beginning is used for implicit parameters, but that is not the case here, no?
Why not call it just `op`?

Reply:
Good catch! Let me try to fix...

Reply:
Can you re-review? I cleaned up everything :)